src/hardware/addon-protocols/rs232-usb.md

```diff
@@ -4,37 +4,232 @@ permalink: /hardware/addon-protocols/rs232-usb/
-
-2. USB-CDC (Communications Device Class)
-
-### Enabling IOX-USB Data Transfer
+### Special Requirements
+##### Enabling IOX-USB Data Transfer
 
 To enable third-party data communication on the IOX-USB, apply the following custom parameter to the GO device through MyGeotab.
 
 ```xml
 <GoParameters><Parameter Description="Enable USB Data" Offset="164" Bytes="02"/></GoParameters>
 ```
 
-The GO device will automatically upgrade to the ProPlus plan — with all associated charges — once third-party data transfer begins.
+The GO device will automatically upgrade to the ProPlus rate plan once third-party data transfer begins.
+##### IOX-USB Communication Consideration
+ 
+The IOX-USB operates as a USB 2.0 full-speed host. The maximum data transfer rate is 12 Mbit/s. The IOX-USB can use two methods to enumerate a USB device:
+
+- The [Android Open Accessory protocol (*AOA*)](https://source.android.com/devices/accessories/protocol.html). This [sample project](https://github.com/Geotab/android-external-device-example "Android Open Accessory Sample") can be used as a framework.
+- USB-CDC (Communications Device Class)
+
+##### Powering a device using the IOX-NFC
+Both the IOX-USB and the IOX-RS232 can provide power to an Add-On Device. 
+
+- The IOX-USB can provide 5V/1.5A as a power output. 
+- The IOX-RS232 supports 900mA at 12/24V to the external red (power) and black (ground) wires. However, it is not required to power the Add On device using the IOX-RS232.
+
+\* *Note* \- It is possible to detect Go Device Resets by monitoring continuity of the output power voltage.
 
 ## Integration Process
 
-The following process should be followed when integrating a third-party device with the GO device using our Third-Party Data Protocol:
+The following process should be followed when integrating a third-party device with the GO device using our Third-Party Data Protocol.
+
+### Contact Geotab Solutions Engineering
+
+Contact the [Geotab Solutions Engineering team](mailto:soleng@geotab.com) with a detailed integration proposal, this should include:
+
+ - A name for the integration
+ - The interfacing hardware
+ - Data types that will be sent to MyGeotab
+ - Will Status Data be required
+ - Will the integration be bi-directional
+ - Expected timelines for integrating
+ 
+The Solutions Engineering team will respond with follow up questions to define the integration, and assign an External device ID, and any Status Data IDs that would be required. 
+
+An additional resource is the Hardware Integration Toolkit with integration walkthrough. *Linked here*
+
+##### Using Status Data IDs
+
+There is an extensive defined Status Data ID list which can be found at MyGeotab >  **Engine & Maintenance** > **Engine & Device…** > **Diagnostics**. For each piece of information, conversion parameters are also required. Conversions must be in the form as per the table below:
```

Comment:
extensively

Reply:
Thanks Stephen, this will be corrected in a future PR.